app/services/api_service.js

```diff
@@ -30,6 +30,7 @@ const handleApiResponse = (response, successCallback, errorCallback) => {
 const getErrorType = (errorStatus) => {
   switch (parseInt(errorStatus)) {
```

Comment:
@limkimsan consider making this as dictionary hash where we could get the error message by just ERROR_DICTIONARY.get(errorStatus)

and the ERROR_DICTIONARY should be a hash like this: {'422', UNPROCESSABLE_ENTITY, '401': UNAUTHORIZED, ..etc.}

Reply:
Noted bong